src/blockdev.rs

```diff
@@ -809,6 +809,50 @@ pub fn lsblk_single(dev: &Path) -> Result<HashMap<String, String>> {
     Ok(devinfos.remove(0))
 }
 
+fn lsblk_all_and_reread_partition_tables() -> Result<Vec<HashMap<String, String>>> {
+    let mut cmd = Command::new("lsblk");
+    cmd.arg("--noheadings")
+        .arg("--nodeps")
+        .arg("--list")
+        .arg("--paths")
+        .arg("--output")
+        .arg("NAME");
+    let output = cmd_output(&mut cmd)?;
+    for dev in output.lines() {
+        if let Ok(mut fd) = std::fs::File::open(dev) {
+            let _ = reread_partition_table(&mut fd);
```

Comment:
Why are we rereading the partition table at all?  The install path already rereads the partition table for the device we installed to.  Ignition does the same (via sgdisk) when modifying partition tables.  For any devices on the system whose partition tables have been modified by other programs, that program is responsible for initiating a reread.  I don't think we should do this unless there's a clear need.

Reply:
on `s390x zVM` i hit an error when using one disk with several paths (multipath was off). `/dev/sdaX`path had new `UUID` after `ignition-ostree-firstboot-uuid`, but `/dev/sdbX/` path had old `UUID`

Reply:
Ah, okay.  That's not an obvious case, so let's add a comment.